packages/core/src/common/message-service-protocol.ts

```diff
@@ -42,9 +42,7 @@ export interface ProgressMessage extends Message {
 export namespace ProgressMessage {
     export const Cancel = 'Cancel';
     export function isCancelable(message: ProgressMessage): boolean {
-        return !message.options
-            || message.options.cancelable === undefined
-            || message.options.cancelable === true;
+        return !!message.options && !!message.options.cancelable;
```

Comment:
@akosyakov would that require any change log as it changes the default behavior in theory?

Reply:
@akosyakov @AlexTugarev 
It's been few days since last comment here.
What should be done to promote this pr?